assets/test/integration/rest_api/project.js

```diff
@@ -3,33 +3,41 @@
 
 require('jasmine-beforeall');
 var h = require('../helpers.js');
+var settingsPage = require('../page_objects/settings.js');
+var buildsPage = require('../page_objects/builds.js');
+var deploymentsPage = require('../page_objects/deployments.js');
+var eventsPage = require('../page_objects/events.js');
+var imagesPage = require('../page_objects/images.js');
+var podsPage = require('../page_objects/pods.js');
+var servicesPage = require('../page_objects/services.js');
+var createProject = require('../page_objects/createProject.js');
+var addToProject = require('../page_objects/addToProject.js');
 
 var goToAddToProjectPage = function(projectName) {
-  var uri = '/project/' + projectName + '/create';
-  h.goToPage(uri);
-  expect(element(by.cssContainingText('h1', "Create Using Your Code")).isPresent()).toBe(true);
-  expect(element(by.cssContainingText('h1', "Create Using a Template")).isPresent()).toBe(true);
-  expect(element(by.model('from_source_url')).isPresent()).toBe(true);
-  expect(element(by.cssContainingText('.catalog h3 > a', "ruby-helloworld-sample")).isPresent()).toBe(true);
+  addToProject.visit();
+  addToProject.expectHeading('Create Using Your Code');
+  addToProject.expectHeading('Create Using a Template');
+  addToProject.expectFromSourceUrl();
+  addToProject.expectTemplate('ruby-helloworld-sample');
+  addToProject.expectBuilderImage('ruby:2.0');
```

Comment:
I'll prob keep this PR to only factoring current running tests to new format.  But I'm motivated to keep our test rolling so hopefully will fast follow getting these others running again :)